ccloud/beginner-cloud/start.sh

```diff
@@ -89,10 +89,12 @@ ccloud environment use $ENVIRONMENT
 # Create a new Kafka cluster and specify it as the default
 ##################################################
 
-CLUSTER_NAME="demo-kafka-cluster"
+CLUSTER_NAME="${CLUSTER_NAME:-demo-kafka-cluster}"
```

Comment:
@rspurgeon  should these 3 parameters also be documented in the README?

Reply:
Docs added